cmd/tendermint/commands/run_node.go

```diff
@@ -35,27 +35,27 @@ var (
 func init() {
 
 	// configuration options
-	runNodeCmd.Flags().StringVar(&moniker, "moniker", config.GetString("moniker"),
```

Comment:
What's the logic surrounding changing all of the default flag values from dependent on the config to hard-coded?